src/components/AppNavHeader.tsx

```diff
@@ -65,13 +78,35 @@ export function AppNavHeader({
           style={styles.backAction}
         />
       ) : null}
+      {options.headerLeft && (
+        <View>
+          {options?.headerLeft?.({
+            tintColor: options.headerTintColor,
+            canGoBack: !!back,
+          })}
+        </View>
+      )}
       <Appbar.Content
-        title={<Title text={title} style={titleStyles} />}
+        title={
+          headerFunction ? (
+            headerFunction({
+              children: title,
+              tintColor: options.headerTintColor,
+            })
+          ) : (
+            <Title text={title} style={titleStyles} />
+          )
+        }
         style={[styles.contentView, back && styles.contentViewWithBackButton]}
       />
-      {options.headerRight?.({
-        canGoBack: !!back,
-      })}
+      {options.headerRight && (
+        <View>
```

Comment:
Just curious, why was this extra View needed?  Another approach could be, if/when it is needed, it's provided inside the `headerRight` func provided.  But I'm guessing we're betting this View will be needed 99% of the time?

Reply:
I had intended to do something else with this view that I never did implement, I can probably drop it.